templates/bundles/TwigBundle/Exception/error500.html.twig

```diff
@@ -15,6 +15,9 @@
 {% block body %}
     <div id="content" class="section section--top section--muted">
         <p>{{'errors.server500.title'|trans}}</p>
-        <small>{{'errors.server500.description'|trans}}</small>
+        <small>{{ 'errors.server500.description'|trans({
+                '%link_start%': '<a href="https://fedidb.org/software/mbin">',
```

Comment:
my only concerns are linking to a domain not under the control of the instance owner. Not that I imagine dansup would ever do something malicious, but if anything happened to the domain or it was ever released, there is a risk involvement there, not sure how instance owners feel about it

Reply:
Technically it's under the control of the instance owner if something is really wrong with this domain, also the owner can still change the url by just changing this template file now and updating the URL.

I'm personally fine for now to link to this domain of Daniel for a list of Mbin instances.

Reply:
Ps. Feel free to give it a follow up pr introducing an admin panel option and db record.